docs/glossary.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Glossary
+sidebar_position: 90
+title: Glossary
+description: Kubewarden glossary
+keywords: [kubewarden, glossary]
+doc-persona: [kubewarden-all]
+doc-type: [explanation]
+doc-topic: [glossary]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/glossary"/>
+</head>
+
+## A
+
+### AdmissionPolicy
+
+A namespace-wide resource. The policy processes only those requests targeting
+the namespace in which the AdmissionPolicy is defined.
+
+## C
+
+### ClusterAdmissionPolicy
+
+A ClusterAdmissionPolicy defines how policies evaluate requests.
+
+### ClusterPolicyReport
+
+A [PolicyReport](#policyreport) and a ClusterPolicyReport store results of
+policy scans. Which one is used depends on the scope of the resource.
+
+## K
+
+### kwctl
+
+A CLI tool allowing administrators to test policies before applying them to a
+cluster.
```

Comment:
The tools also allows the user to generate the Kubernetes yaml files used to deploy the policies.